extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -361,5 +369,53 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>spark-3.1</id>
```

Comment:
The dedicated profiles for Spark may be necessary.

Reply:
what should I do?